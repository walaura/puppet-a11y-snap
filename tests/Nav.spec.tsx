import React from "react";
import Nav from "./Nav";
import getSnap, { utils } from "../src/index";

const { getRegions } = utils;
test("Link changes the class when hovered", async () => {
  const component = <Nav />;
  const snap = await getSnap(component);
  const regions = getRegions(snap);

  expect(snap).toMatchSnapshot();
  expect(regions.length).toBe(1);
});
