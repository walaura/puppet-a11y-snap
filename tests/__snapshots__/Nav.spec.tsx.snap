// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
Array [
  Object {
    "children": Array [
      Object {
        "children": Array [
          Object {
            "children": Array [
              Object {
                "children": Array [
                  Object {
                    "name": "•",
                    "role": "text",
                  },
                ],
                "name": "•",
                "role": "ListMarker",
              },
              Object {
                "children": Array [
                  Object {
                    "name": "wololo",
                    "role": "text",
                  },
                ],
                "name": "wololo",
                "role": "link",
              },
            ],
            "name": "",
            "role": "listitem",
          },
          Object {
            "children": Array [
              Object {
                "children": Array [
                  Object {
                    "name": "•",
                    "role": "text",
                  },
                ],
                "name": "•",
                "role": "ListMarker",
              },
              Object {
                "children": Array [
                  Object {
                    "name": "wololo",
                    "role": "text",
                  },
                ],
                "name": "wololo",
                "role": "link",
              },
            ],
            "name": "",
            "role": "listitem",
          },
          Object {
            "children": Array [
              Object {
                "children": Array [
                  Object {
                    "name": "•",
                    "role": "text",
                  },
                ],
                "name": "•",
                "role": "ListMarker",
              },
              Object {
                "children": Array [
                  Object {
                    "name": "wololo",
                    "role": "text",
                  },
                ],
                "name": "wololo",
                "role": "link",
              },
            ],
            "name": "",
            "role": "listitem",
          },
          Object {
            "children": Array [
              Object {
                "children": Array [
                  Object {
                    "name": "•",
                    "role": "text",
                  },
                ],
                "name": "•",
                "role": "ListMarker",
              },
              Object {
                "children": Array [
                  Object {
                    "name": "wololo",
                    "role": "text",
                  },
                ],
                "name": "wololo",
                "role": "link",
              },
            ],
            "name": "",
            "role": "listitem",
          },
        ],
        "name": "",
        "role": "list",
      },
    ],
    "name": "Navigation",
    "role": "region",
  },
]
`;
